### PR TITLE
Add "Edit this page" link everywhere except /

### PR DESCRIPTION
**Pull request by handlers**
This PR adds an 'Edit this page' link everywhere on the [Otto website](https://ottoproject.io/) except the homepage.

I have also added a `Makefile` to the `website` directory and updated the README accordingly, and have updated the `Gemfile.lock` file to point to the latest `middleman-hashicorp` gem.

### CODE REVIEW

File: website/Makefile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+all: build
+
+init:
+	bundle
+
+dev: init
+	bundle exec middleman server
+
+build: init
+	bundle exec middleman build
```
Comment (justincampbell) [inline website/Makefile]: Can you add `.PHONY` for these?

**Comment by handlers** [comment]
Made some `Makefile` changes, h/t @justincampbell

**Comment by justincampbell** [comment]
:+1:

**Comment by mitchellh** [comment]
Looks amazing! Thanks!

**Pull request merged**